metron-platform/metron-integration-test/src/main/resources/sample/config/sensors/websphere.json

```diff
@@ -0,0 +1,22 @@
+{
+  "index": "websphere",
+  "batchSize": 1,
+  "enrichmentFieldMap":
```

Comment:
So, this structure got refactored a bit as of the last PR.  Check out the example [here](https://github.com/apache/incubator-metron/tree/master/metron-platform/metron-common#example)  The upshot is that instead of `enrichmentFieldMap` and `threatIntelFieldMap` there are `enrichment` and `threatIntel` sections and `fieldMap` is in both.